### ISSUE TITLE
About Wan2.1 VAL

### ISSUE THREAD

**Issue by Weixiang-Sun** [bghira/SimpleTuner#1530]
Hi dear team,

when I use full finetune on wan2.1 for t2v with validation_prompt, there's **_[ERROR] Error logging validation images: 'list' object has no attribute 'ndim'_**

the following is my config

`{
    "--resume_from_checkpoint": "latest",
    "--data_backend_config": "config/multidatabackend.json",
    "--aspect_bucket_rounding": 2,
    "--seed": 42,
    "--minimum_image_size": 0,
    "--disable_benchmark": false,
    "--output_dir": "xxxxxx",
    "--use_ema": "true",
    "--num_train_epochs": "1",
    "--max_train_steps": 0,
    "--checkpointing_steps": 500,
    "--checkpoints_total_limit": 5,
    "--attention_mechanism": "diffusers",
    "--tracker_project_name": "echora",
    "--tracker_run_name": "simpletuner-full",
    "--report_to": "wandb",
    "--model_type": "full",
    "--pretrained_model_name_or_path": "xxxxxx",
    "--model_family": "wan",
    "--train_batch_size": 1,
    "--gradient_checkpointing": "true",
    "--caption_dropout_probability": 0.0,
    "--resolution_type": "pixel_area",
    "--resolution": "112",
    "--validation_seed": 42,
    "--validation_steps": "10",
    "--validation_resolution": "112x112",
    "--validation_guidance": 4.0,
    "--validation_guidance_rescale": "0.0",
    "--validation_num_inference_steps": "20",
    "--validation_prompt": "MY_PROMPT",
    "--mixed_precision": "bf16",
    "--optimizer": "adamw_bf16",
    "--learning_rate": "1e-6",
    "--lr_scheduler": "polynomial",
    "--lr_warmup_steps": 0,
    "--validation_torch_compile": "false",
    "--override_dataset_config": "true"
}`
